index.html

```diff
@@ -15,16 +15,20 @@ <h1><span class="bold magenta"><strong>NUMBER</strong></span>GUESSER</h1>
       <div class="container">
         <section class="side left-side">
           <section class="game-box">
-            <h4 >Set Range</h4>
-              <section class="range-box">
-                <section>
-                  <label for="min-range">Min Range</label>
-                  <input id="min-range" type="number" class="" name="minRange"/></section>
-                  <section>
-                  <label for="max range">Max Range<label><input type="number" class="" name="maxRange"/>
-                </section>
-            <button class="game-button">UPDATE</button>
-            </section>
+            <h4 class="range-input">Set Range</h4>
+            <div class="range-box">
+              <section class="content-box">
+                  <label class="range-input" for="min-range">Min Range</label>
+                <input id="min-range" type="number" class="range-input" name="minRange"/>
+              </section>
+              <section class="content-box max-range">
```

Comment:
We are using max-range for both an id and class lets use something different here. We decided on max-box